Add Instructions to disable swipe-to-dismiss on WearOS (#10422)
godotengine/godot-docs: tutorials/export/android_gradle_build.rst

    For example, ``_example/image.png`` will **not** be included as an asset,
    but ``_image.png`` will.

> Disable Swipe-to-Dismiss on Wear OS
> -----------------------------------

Reviewer: ```suggestion
Disabling Swipe-to-Dismiss on Wear OS
-------------------------------------
```
Slightly more idiomatic format for titles

Also, *possibly* also this. The current style guide for the manual is to avoid Actual Title Case for header titles, and only capitalize proper nouns and the first word. Seems like people refer to "swipe to dismiss" several ways, but here is [one instance](https://developer.android.com/training/wearables/views/exit#swipe-to-dismiss) of the official android docs using "swipe-to-dismiss"
```suggestion
Disabling swipe-to-dismiss on Wear OS
-------------------------------------
```

To disable swipe-to-dismiss functionality on Wear OS devices, follow these steps:

> 1. Open the file ``res://android/build/res/values/themes.xml`` in your file editor.

Reviewer: ```suggestion
1. Open the file ``res://android/build/res/values/themes.xml`` in a text editor.
```
Possibly this?

> 2. Add or update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the `GodotAppMainTheme` style.

Reviewer: ```suggestion
2. Add or update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the ``GodotAppMainTheme`` style.
```
Like this if you want `code`

```suggestion
2. Add or update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the *GodotAppMainTheme* style.
```
Like this if you actually want *italics*

In RST single backticks does "default style" which is a link/reference if you also put _ at the end, and italics otherwise

Also, it's currently slightly ungrammatical, since "Add the attribute to false" isn't grammatical. But the grammatically correct options both seem a bit more awkward, so IDK, it's maybe fine as-is:
```suggestion
2. Add the ``android:windowSwipeToDismiss`` attribute in ``GodotAppMainTheme`` style if you don't already have it. Then set it to ``false``.
```
```suggestion
2. Update the ``android:windowSwipeToDismiss`` attribute to ``false`` in the `GodotAppMainTheme` style. If you don't already have it, add it first.
```

Example:

.. code-block:: xml

    <style name="GodotAppMainTheme" parent="@android:style/Theme.DeviceDefault.NoActionBar">
        <item name="android:windowDrawsSystemBarBackgrounds">false</item>

>         <item name="android:windowSwipeToDismiss">false</item>

Reviewer 2: @syntaxerror247 We could add an advanced export option to automatically update this value for gradle builds.

Author: @m4gr3d I initially considered this approach but decided to document instructions instead, as it seemed feasible only for Gradle export. However, after seeing your other comment, if it's acceptable to have Gradle-specific build options, I can close this PR and open a new one in the Godot repository to implement it.

Do you think adding this to the project settings in `pointing` would be a better solution?
